make/rules.mk

```diff
@@ -84,6 +84,15 @@ flash: $(OBJ_DIR)/flash
 
 upload: $(OBJ_DIR)/upload
 
+reboot_to_dfu:
+	stty -f /dev/cu.usbmodemflip_* 115200
```

Comment:
It is either cross platform or it's not here.

Just keep a bash script for this purpose